src/hardware/input/mouseif_dos_driver.cpp

```diff
@@ -1444,7 +1522,7 @@ static Bitu int33_handler()
 		// Both buttons down = pen pressed, otherwise pen considered
 		// off-screen
 		// TODO: maybe implement light pen using SDL touch events?
-		LOG(LOG_MOUSE, LOG_ERROR)("Mouse light pen emulation not implemented");
+		LOG_WARNING("MOUSE (DOS): Light pen emulation not implemented");
```

Comment:
Nice normalisation of log messages. 🎉  Me approves 😎 🤘🏻

Reply:
Actually, I did a small change here - there is no warning log if the application just disables the light pen emulation; _The Settlers_ does this during the startup.